src/context/EntryContext.jsx

```diff
@@ -0,0 +1,19 @@
+import { createContext, useState, useContext } from "react";
+
+const EntryContext = createContext();
+const EntryProvider = ({children}) => {
+    const [entries, setEntries] = useState([]);
+    return (
+        <EntryContext.Provider value={{entries, setEntries}}>{children}</EntryContext.Provider>
+    )
```

Comment:
Be careful with your formatting here. I don't know if this is an issue on GitHub or not, but the formatting of your return in this function is a little hard to read here.